Kickassanime & Hianime: Fix subtitles

V3u47ZoN opened the pull request:
Checklist:

- [x] ~~Updated extVersionCode value in build.gradle for individual extensions~~ handled by lib bump
- [x] ~~Updated overrideVersionCode or baseVersionCode as needed for all multisrc extensions~~ handled by lib bump
- [x] ~~Referenced all related issues in the PR body (e.g. "Closes #xyz")~~ no report on github, only on discord
- [x] Added the isNsfw = true flag in build.gradle when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the id if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed web_hi_res_512.png when adding a new extension
- [x] Have made sure all the icons are in png format

If the cue text contained empty lines then subtitles would fail to show up: https://discord.com/channels/1253381712306634904/1253387549246754937/1363874613611855962 & https://discord.com/channels/1253381712306634904/1253387549246754937/1364456864011784203

Example of broken subtitles:
Kickassanime: death note episode 36
Hianime: witch watch episode 3 @ 17:20

Arkai1 commented:
Run the checks and the person has also confirmed that it is Aniyomi's problem working fine in Dantotsu

Arkai1 reviewed (commented):
If you have made changes in Aniplay.kt, have you updated the version code in build.gradle

Arkai1 commented:
Also you don't need do these changes as subtitles are showing haven't you read full conversation 
https://discord.com/channels/1253381712306634904/1253387549246754937/1363880181357613109

Arkai1 commented:
> > If you have made changes in Aniplay.kt, have you updated the version code in build.gradle. Update version code in build.gradle so it will not conflict with current build
> 
> back before dmca updating a lib would automatically bump all extensions using it is that not the case anymore ?

👍 Then no issue

Arkai1 reviewed (commented):
Leave the version code to 5 for zoro theme as there is no need for updating its base version code

V3u47ZoN commented on lib-multisrc/zorotheme/build.gradle.kts:
So will extensions be bumped or not whenever a lib changes? If so, i wouldnt need to manually bump anything no?

V3u47ZoN commented:
Ok i fixed the codes now, since aniplay and kickassanime relies on playlist-utils, which was changed, theres no need to manually bump the version just like with zorotheme. I hope you can merge this as soon as possible, thank you.

V3u47ZoN commented:
> > Ok i fixed the codes now, since aniplay and kickassanime relies on playlist-utils, which was changed, theres no need to manually bump the version just like with zorotheme. I hope you can merge this as soon as possible, thank you.
> 
> Ask GregiPlays or hak in the dev chat of discord and complete the checks

I have done all the checks, i just removed the ones that arent relevant here, i can add them back if you like. Also, the discord server requires a verified phone number to speak so i cannot talk there.

Arkai1 commented:
> > > Ok i fixed the codes now, since aniplay and kickassanime relies on playlist-utils, which was changed, theres no need to manually bump the version just like with zorotheme. I hope you can merge this as soon as possible, thank you.
> > 
> > 
> > Ask GregiPlays or hak in the dev chat of discord and complete the checks
> 
> I have done all the checks, i just removed the ones that arent relevant here, i can add them back if you like. Also, the discord server requires a verified phone number to speak so i cannot talk there.

Have you run the pr checks jobs

V3u47ZoN commented:
> > > > Ok i fixed the codes now, since aniplay and kickassanime relies on playlist-utils, which was changed, theres no need to manually bump the version just like with zorotheme. I hope you can merge this as soon as possible, thank you.
> > > 
> > > 
> > > Ask GregiPlays or hak in the dev chat of discord and complete the checks
> > 
> > 
> > I have done all the checks, i just removed the ones that arent relevant here, i can add them back if you like. Also, the discord server requires a verified phone number to speak so i cannot talk there.
> 
> Have you run the pr checks jobs

A maintainer needs to approve it, theres nothing i can do except wait for someone to approve

Arkai1 commented:
> > > > > Ok i fixed the codes now, since aniplay and kickassanime relies on playlist-utils, which was changed, theres no need to manually bump the version just like with zorotheme. I hope you can merge this as soon as possible, thank you.
> > > > 
> > > > 
> > > > Ask GregiPlays or hak in the dev chat of discord and complete the checks
> > > 
> > > 
> > > I have done all the checks, i just removed the ones that arent relevant here, i can add them back if you like. Also, the discord server requires a verified phone number to speak so i cannot talk there.
> > 
> > 
> > Have you run the pr checks jobs
> 
> A maintainer needs to approve it, theres nothing i can do except wait for someone to approve

Will ask GregiPlays to check it

krysanify commented:
> since aniplay and kickassanime relies on playlist-utils

there are more than these two that use playlist-utils tho i don't know how many actually extract subtitles. consider testing on those exts as well (assuming that they were working before these changes)

V3u47ZoN commented:
> > since aniplay and kickassanime relies on playlist-utils
> 
> there are more than these two that use playlist-utils tho i don't know how many actually extract subtitles. consider testing on those exts as well (assuming that they were working before these changes)

Since the function isn't actually used in playlist-utils but rather something that the extensions have to opt-in to use its not going to affect any other extension
